modulefiles/srw_common

```diff
@@ -20,9 +20,9 @@ module load ip/3.3.3
 module load sp/2.3.3
 module load w3nco/2.4.1
 
-module load-any gftl-shared/v1.3.3 gftl-shared/1.3.3
+module load-any gftl-shared/v1.5.0 gftl-shared/1.5.0
```

Comment:
@chan-hoo For any given software, how do you find out what version numbers to update to?

Reply:
Ah ok, thanks.